op-deployer/pkg/deployer/flags.go

```diff
@@ -52,15 +52,12 @@ func NewDeploymentTarget(s string) (DeploymentTarget, error) {
 	}
 }
 
-var DefaultCacheDir string
-
-func init() {
-	var err error
+func GetDefaultCacheDir() string {
 	homeDir, err := os.UserHomeDir()
 	if err != nil {
-		panic(fmt.Sprintf("failed to get home directory: %s", err))
+		return ".op-deployer/cache" // Fallback to local directory
```

Comment:
I think we should do one of the following here:
- Still panic if HOMEDIR can't be determined.
- At least log (log.Println - Go std log lib) about falling back to the `local directory due to the failure with determining home dir: %s, err.Error()`

Reply:
Added log in https://github.com/ethereum-optimism/optimism/pull/14810/commits/3c4ce1997b4dfac354ebd7c0ad744474ecbc9a55